test/pool-test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('./util/assert');
+const FullNode = require('../lib/node/fullnode');
+const SPVNode = require('../lib/node/spvnode');
+
+const full = new FullNode({
+  network: 'regtest'
+});
+
+const spv = new SPVNode({
+  network: 'regtest'
+});
+
+const dummyHash = "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef";
+
+describe('Pool', function() {
+  it('should not have nonexistent txs when there is no mempool', () => {
+    assert(!spv.pool.hasTX(dummyHash));
+    assert(!spv.pool.hasTX(dummyHash));
```

Comment:
This code looks like a mistake, but it isn't. I recommend making it clear in the `it` description like so: "should check if a tx exists without adding it to the filter".

Reply:
Done